sherpa-onnx/python/csrc/faked-tts.cc

```diff
@@ -0,0 +1,47 @@
+// sherpa-onnx/python/csrc/faked-tts.cc
+//
+// Copyright (c)  2025  Brno University of Technology (author: Karel Vesely)
+
+#include "sherpa-onnx/python/csrc/offline-tts-kokoro-model-config.h"
+#include "sherpa-onnx/python/csrc/offline-tts-matcha-model-config.h"
+#include "sherpa-onnx/python/csrc/offline-tts-model-config.h"
+#include "sherpa-onnx/python/csrc/offline-tts-vits-model-config.h"
```

Comment:
can we remove these lines?

Reply:
I am not sure. I tried to keep the same `.h` files and replace the `.cc` files.
So these are irrelevant for the pybind11 API for python ?

Reply:
I tried it, and rebuilt with a new `buid` dir. The error is still the same:
```
ImportError: cannot import name 'FastClustering' from '_sherpa_onnx' (/mnt/matylda5/iveselyk/CNECT_TENDER/SHERPA_ONNX/CONDA_ENVIRONMENT/lib/python3.9/site-packages/_sherpa_onnx.cpython-39-x86_64-linux-gnu.so)
```